rewardsPointsSystem.py

```diff
@@ -34,15 +34,16 @@
 
 """
 
+from collections import defaultdict # fix: imports at the top
+
 class Item:
   def __init__(self, itemId, item_price):
     self.itemId = itemId
-    self.item_price = item_price
-
-from collections import defaultdict    
+    self.item_price = item_price 
```

Comment:
I would suggest consolidating variable names to the same type (camelCase, snake_case, etc)!

Reply:
done

Reply:
Nice job addressing the redundant/unnecessarily long code!